### PR TITLE
Add styling to components

### PR DESCRIPTION
**Pull request by Dinma-Faith-zz**
**In this final milestone I did the following**
 
 * Style Project UI following  [Zipline Design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek.md)
 * Apply React best practices and language style guides in code.

### CODE REVIEW

File: src/components/book.js [hunk 1]
```diff
@@ -12,26 +14,59 @@ const Book = (props) => {
...
+        <li />
+      </ul>
+      <li className="lastCol">
+        <p className="currChapter">CURRENT CHAPTER</p>
+        <p id="chapter">
+          Chapter
+          {' '}
+          {chapterVal}
+        </p>
+        <button type="button" className="update-btn">UPDATE PROGRESS</button>
```
[the remark this answers is not in the document]
Comment (medaminedev66) [reply src/components/book.js:65]: - [ ] As requested by the previous reviewer:
- Kindly use the same font used in the template.
- Kindly add a border-radius around the 'Update progress' button.

File: src/components/book.js [hunk 2]
```diff
@@ -12,26 +14,58 @@ const Book = (props) => {
...
+              Delete
+            </button>
+            <button type="button" className="text button">Edit</button>
+          </div>
+        </li>
+      </ul>
+
+      <ul className="secondCol">
+        <li>
+          <div style={{ width: 100, height: 100 }}>
```
Comment (medaminedev66) [inline src/components/book.js]: - [ ] [REQUIRED] As requested by the previous reviewer, Kindly avoid inline styling. It is one of CSS bad practices. When you add inline styles, it makes it difficult revising the design of your site and makes your code harder to manage.
I would recommend moving the style out of the `div`. Try to define a constant `style` in the `Book` component and give it to the style attribute in the div.

File: src/components/form.js [hunk 3]
```diff
@@ -20,27 +20,38 @@ function Form() {
...
+          <option value="" hidden>
+            Category
+          </option>
+          <option value="Dramma">Dramma</option>
+          <option value="Documentary">Documentary</option>
+          <option value="Fiction">Fiction</option>
+          <option value="Crime">Crime</option>
+          <option value="Romance">Romance</option>
+        </select>
+        <button className="btn" type="submit">Add Book</button>
```
Comment (medaminedev66) [inline src/components/form.js]: - [ ] As requested by the previous reviewer:
- [OPTIONAL] Kindly ensure the border around the input field is the same color as the one in the Zeplin design
- [REQUIRED] Kindly align the input fields and the button as it is aligned in the template.

File: src/components/book.js [hunk 4]
```diff
@@ -12,26 +14,58 @@ const Book = (props) => {
...
+          </p>
+          <div className="button-text">
+            <button type="button" className="text button">Comment</button>
+            <button type="button" className="text button" onClick={handleDelete}>
+              Delete
+            </button>
+            <button type="button" className="text button">Edit</button>
+          </div>
+        </li>
+      </ul>
```
Comment (medaminedev66) [inline src/components/book.js]: [OPTIONAL] As requested by the previous reviewer, kindly use the font family provided in the template for all text.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Dinma-Faith 😃,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Your design looks great 👏

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keneogu — APPROVED** [review]
> ## Hi @Dinma-Faith,
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> 
> ## Congratulations!     
> Cheers and Happy coding!clapclapclap
> 
> image [ ](https://camo.githubusercontent.com/2366be437d35bce33dd123620530e9131a8ac147fe960e76a36614dc703e8e22/68747470733a2f2f692e70696e696d672e636f6d2f6f726967696e616c732f37372f30652f62302f37373065623035373434393538326334343637326232643264316363333962362e676966) image [ ](https://camo.githubusercontent.com/2366be437d35bce33dd123620530e9131a8ac147fe960e76a36614dc703e8e22/68747470733a2f2f692e70696e696d672e636f6d2f6f726967696e616c732f37372f30652f62302f37373065623035373434393538326334343637326232643264316363333962362e676966)
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**